src/qautostart.cpp

```diff
@@ -31,102 +31,86 @@
 #include <QFile>
 #include <QDir>
 
-
-Autostart::Autostart() {
-
-}
-
 #if defined (Q_OS_WIN)
 #define REG_KEY "HKEY_CURRENT_USER\\Software\\Microsoft\\Windows\\CurrentVersion\\Run"
 
-bool Autostart::isAutostart() const {
+bool Autostart::isAutostart() const
+{
     QSettings settings(REG_KEY, QSettings::NativeFormat);
 
-    if (settings.value(appName()).isNull()) {
-        return false;
-    }
-
-    return true;
+    return !settings.value(appName()).isNull();
 }
 
-void Autostart::setAutostart(bool autostart) {
+void Autostart::setAutostart(bool autostart)
+{
     QSettings settings(REG_KEY, QSettings::NativeFormat);
 
-    if (autostart) {
-        settings.setValue(appName() , appPath().replace('/','\\'));
-    } else {
-        settings.remove(appPath());
-    }
+    if (autostart) settings.setValue(appName(), appPath().replace('/','\\'));
```

Comment:
Same here. Moreover don't you think curly braces makes the code more readable?

Reply:
For a single line if, I don't prefer to use braces. I think it looks better that way. I'll add back if you want, not a big deal.